composer.json

```diff
@@ -3,7 +3,7 @@
     "description": "Helps to discover and install tools",
     "type": "project",
     "require": {
-        "php": "~8.1.0 || ~8.2.0 || ~8.3.0",
```

Comment:
Looking at this code and the one in devkit, we could extract the supported PHP versions from composer.json here instead of hardcoding it, probably also generate the Github actions YAML file.